contract/src/lib.rs

```diff
@@ -22,6 +26,10 @@ pub struct ParticipantInfo {
     pub id: ParticipantId,
     pub account_id: AccountId,
     pub url: String,
+    /// The public key used for encrypting messages.
+    pub cipher_pk: hpke::PublicKey,
+    /// The public key used for verifying messages.
```

Comment:
I would rather have longer names here and in other places, like `msg_encryption_pk`, `msg_signature_pk`. People and other developers can confuse these with key shares.